package.json

```diff
@@ -51,7 +51,9 @@
     "semver": "5.1.0",
     "shelljs": "0.6.0",
     "webpack": "1.12.14",
+    "webpack-dev-middleware": "1.5.1",
```

Comment:
Historically I've found it's useful to have things be explicit and controlled by the package.json more tightly. For similar reasons I've turned off the default '^' prefix in .npmrc. Although you're still at upstream's mercy. I've seen several occasions where things have broken due to implicit upgrades of deps.